src/components/head-metadata/index.tsx

```diff
@@ -44,23 +45,26 @@ export default function HeadMetadata(props: HeadMetadataProps) {
 
 	return (
 		// TODO: OpenGraph image to be passed as the image prop here
-		<HashiHead
-			title={title}
-			siteName={title}
-			pageName={title}
-			description={finalDescription}
-			image={ogImageUrl}
-		>
-			<link rel="icon" href="/favicon.ico" sizes="any" />
-			<link rel="icon" href="/icon.svg" type="image/svg+xml" />
-			<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-			<meta
-				name="google-site-verification"
-				content="zRQZqfAsOX-ypXfU0mzAIzb5rUvj5fA4Zw2jWJRN-JI"
-			/>
+		<>
+			<SelfReferentialCanonicalTag />
```

Comment:
One note is that the tutorial view pages already have canonicals set because the content is duplicated across collections. Can we adjust this to be optional or make sure the tutorial views don't get this self-referential tag?

Reply:
FWIW, I _think_ this might already be handled by `next/head`, it should dedupe tags and prioritize those which are rendered deeper in the react tree.

Reply:
This should allow for deduping - https://github.com/hashicorp/dev-portal/pull/969/commits/fef495a8670901cd6b0790b291fcf1b469b49b90